src/components/CountUp/index.tsx

```diff
@@ -2,13 +2,21 @@ import usePrevious from '../../hooks/usePrevious'
 import React from 'react'
 import { CountUp as CountUpImpl } from 'use-count-up'
 
-export default function CountUp({ enabled, value }: { enabled: boolean; value: number }) {
+export default function CountUp({
+  enabled,
+  value,
+  decimalPlaces
+}: {
+  enabled: boolean
+  value: number
+  decimalPlaces?: number
+}) {
   const previousValue = usePrevious(value)
 
   return (
     <CountUpImpl
       isCounting={enabled}
-      decimalPlaces={4}
+      decimalPlaces={decimalPlaces ?? 4}
```

Comment:
Add a global const above `export default function CountUp({` called `DEFAULT_DECIMAL_PLACES` and then set that as the default prop value:

```js
export default function CountUp({
  enabled,
  value,
  decimalPlaces = DEFAULT_DECIMAL_PLACES
}: {
  enabled: boolean
  value: number
  decimalPlaces: number
}) {
```

```suggestion
      decimalPlaces={decimalPlaces}
```